package.json

```diff
@@ -23,18 +24,18 @@
     "repository": "github:cloudflare/voprf-ts",
     "devDependencies": {
         "@peculiar/webcrypto": "^1.2.3",
```

Comment:
Please can we switch to removing the caret for version numbers. I strongly support manual updates somewhat frequently though!